### PR TITLE
Fix #1199 - Blank screen in Account Activity

### LINKED ISSUE #1199: Blank Screen in Accounts Activity
**Summary:**

Whenever  the filter in accounts activity returns empty list ( no result) , then there's just a blank screen in the view . This is affects the user experience

**Steps to reproduce:**

Go to Account Activity ->  Apply filters that  give no result 

**Expected behaviour:**

Should have displayed an appropriate message 

**Observed behaviour:**

Blank Screen

**Device and Android version:**

Xiaomi Poco F1
Android 9
No custom rom

**Screen Recording**

Screenrecorder-2019-10-09-22-36-47-686

### PR DESCRIPTION
**Pull request by devansh-299**
Fixes #1199 

Screen Recording

fix1199

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -514,6 +514,11 @@
 
     </string-array>
 
+    <string name="no_saving_account">No Saving Accounts to show</string>
+    <string name="no_loan_account">No Loan Accounts to show</string>
+    <string name="no_sharing_account">No Sharing Accounts to show</string>
+
+
```
Comment (geekanamika) [inline app/src/main/res/values/strings.xml]: One line space will suffice, you can remove on extra line after strings.

**Review by geekanamika — CHANGES_REQUESTED** [review]
I would recommend using "showSweetCustomErrorUI" method from SweetUIErrorHandler to display appropriate image & message when filter search results with no data. No need to add addition imageview & text, handle it's visibility throughout the code. 
Can you please update the PR?

**Comment by devansh-299** [comment]
@geekanamika , I updated the PR as requested but the Travis build failed stating the cause as *java.lang.IllegalArgumentException: Unable to exclude rules [GuardLogStatement];*

**Comment by devansh-299** [comment]
@geekanamika  I have updated the PR , please review it

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#Tue Oct 10 11:28:53 IST 2017
+#Thu Oct 17 16:49:19 IST 2019
```
Comment (geekanamika) [inline gradle/wrapper/gradle-wrapper.properties]: remove comment changes here.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -54,62 +54,62 @@
             android:windowSoftInputMode="adjustResize"/>
 
         <activity android:configChanges="orientation|screenSize"
-                  android:launchMode="singleTask"
-                  android:name="org.mifos.mobile.ui.activities.PassCodeActivity"
-                  android:windowSoftInputMode="adjustResize"/>
+            android:launchMode="singleTask"
```
Comment (geekanamika) [inline app/src/main/AndroidManifest.xml]: There's no change in this file, why format & add it in a commit? 
Please remove this file from the commit.

**Review by geekanamika — CHANGES_REQUESTED** [review]
Added two comments. 
For future PRs, Please don't add any file which doesn't have any relevant changes related to the PR/issue. Thanks :)

**Comment by devansh-299** [comment]
> Added two comments.
> For future PRs, Please don't add any file which doesn't have any relevant changes related to the PR/issue. Thanks :)

Thanks for the advice . I won't repeat this mistake

**Review by geekanamika — APPROVED** [review]

**Pull request merged**